### PR TITLE
Jukebox Popup

### PR DESCRIPTION
**Pull request by OnlyBMan**
Adds the Jukebox popup for PE clients

### CODE REVIEW

File: src/protocolsupport/protocol/typeremapper/pe/PERecord.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import protocolsupport.api.TranslationAPI;
+import protocolsupport.protocol.packet.middleimpl.ClientBoundPacketData;
+import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.Chat;
+import protocolsupport.protocol.serializer.StringSerializer;
+import protocolsupport.protocol.utils.ProtocolVersionsHelper;
+import protocolsupport.protocol.utils.i18n.I18NData;
+
+public class PERecord {
+
+	public static ClientBoundPacketData createPacket(String disc) {
```
Comment (Shevchik) [inline src/protocolsupport/protocol/typeremapper/pe/PERecord.java]: Protocol version and locale needs to be passed and used.

**Comment by OnlyBMan** [comment]
Hopefully this is the proper way to pass those values. 😁

**Comment by Shevchik** [comment]
No. See every other packet create method.

**Comment by OnlyBMan** [comment]
Is this what you meant Shev?

**Comment by Shevchik** [comment]
Yes

**Pull request merged**